styles.css

```diff
@@ -33,7 +34,7 @@ img{
 }
 
 @media(max-width:600px){
-  .main{
+  header, .main, aside, footer{
     background:red;
   }
 }
```

Comment:
Don't see you re-arranging the order of elements for Question #7. You'll want to make use of Flexbox's `order` property on the elements for which you need to re-arrange the order.